src/Webapi/Webapi__Dom/Webapi__Dom__DomStringMap.re

```diff
@@ -0,0 +1,12 @@
+type t = Dom.domStringMap;
+
+type key = string;
+
+[@bs.get_index] [@bs.return nullable]
+external get: (t, key) => option(string) = "";
+let getPipe: (key, t) => option(string) = (key, map) => get(map, key); 
```

Comment:
You can just name this `get`. Then it will shadow the `external`, but allows you to still use it inside the function definition.